### DISCUSSION TITLE
Tap-to-Earn Game Suggestions/Ideas

### DISCUSSION THREAD

**Discussion by tboy1337** [masterking32/MasterCryptoFarmBot#8] [Ideas]
Hamster Kombat: 300,000,000 Users
TapSwap: 50,000,000 Users
Blum: 50,000,000 Users
Notcoin: 40,000,000 Users
Yescoin: 18,000,000 Users
Pixeltap: 15,000,000 Users
Catizen: 10,000,000 Users
Major of Telegram: 3,000,000 Users

There are many more Tap-to-Earn games on [https://chainplay.gg/genre/tap-to-earn/](https://chainplay.gg/genre/tap-to-earn/)

**Comment by copycatcodex** [comment]
i have duckchain https://github.com/copycatcodex/DuckChain-Bot but this is still very raw and nothing new, just click.

**Comment by cookie027** [comment]
according to tokenomics, there is no need to play with season 2.

**Comment by tboy1337** [comment]
> according to tokenomics, there is no need to play with season 2.

Can you share your source?

**Comment by tboy1337** [comment]
https://playtoearn.com/blockchaingames/TON/All-Genre/All-Status/All-Device/All-NFT/All-PlayToEarn/All-FreeToPlay

**Comment by tboy1337** [comment]
https://tgstat.com/ratings/channels
